src/evergreen/metrics/buildmetrics.py

```diff
@@ -315,7 +314,6 @@ def _count_task(self, task: "Task") -> None:
         if task.start_time:
             self._start_times.append(task.start_time)
 
-        self.estimated_build_costs += task.estimated_cost
```

Comment:
bc `task.estimated_cost` is being deprecated, and `self.estimated_build_costs` is 100% dependent on `task.estimated_cost` got rid of this in `BuildMetrics` object.